voluncheer/job_board/views.py

```diff
@@ -3,61 +3,18 @@
 
 from job_board.models import Job
```

Comment:
I think job_board folder has become jobboard folder. The original job_board should be removed, right? It's no longer in develop branch.

Reply:
Not on develop branch by my observation, are you sure you merged it?